expcore/Mini_games.lua

```diff
@@ -871,7 +872,7 @@ function Mini_games.print_results(results, unit, names, limit)
     names = names or {}
     limit = limit or 5
     for i, result in ipairs(results) do
-        if result.place < limit then
+        if result.place and result.place < limit then
```

Comment:
Would it not be an error for an entry to not contain a place?
Also this is might conflicting with the pr #47

Reply:
No it is not an error.  If you finish some laps and wreck the car some too but does not reach the finish line before the race ends it would still be useful to have the lap counts and cars wrecked stored.  Same goes for extra data to production battles and speedruns.  Logic wise it does not conflict with #47, but they do modify the same lines, I'll fix the conflict.

Reply:
Thanks for the clarification, did not see the comment saying that place and score are now optional. I no longer have an issue with this code although there might be issues it other places due to the assumption that they were required.